Tasks/VsTest/Helpers.ps1

```diff
@@ -43,6 +43,33 @@ function IsVisualStudio2015Update1OrHigherInstalled {
 	return $false
 }
 
+function GetResultsLocation {
+	[cmdletbinding()]
+	[OutputType([System.String])]
+	param(		
+		[string]$runSettingsFilePath		
+	)
+
+    if((![System.String]::IsNullOrWhiteSpace($runSettingsFilePath)) -And ([string]::Compare([io.path]::GetExtension($runSettingsFilePath), ".runsettings", $True) -eq 0) -And !(Test-Path $runSettingsFilePath -pathtype container))
+    {
+        $runSettingsForTestResults = [System.Xml.XmlDocument](Get-Content $runSettingsFilePath)
+        $resultsDirElement = $runSettingsForTestResults.SelectNodes("//RunSettings/RunConfiguration/ResultsDirectory")
+		if($resultsDirElement -And $resultsDirElement.Count -ne 0)
+		{
+            $customLocation = $runSettingsForTestResults.RunSettings.RunConfiguration.ResultsDirectory       
+            if([io.path]::IsPathRooted($customLocation))
+            {
+                return $customLocation
+            }
+            else
+            {
+                return [io.path]::GetFullPath([io.path]::Combine([io.path]::GetDirectoryName($runSettingsFilePath), $customLocation))                
```

Comment:
I see that when run in parallel is enabled and user is using relative path we wont work. is it true?